library/external/campaignmonitor.php

```diff
@@ -122,17 +122,17 @@ class CampaignMonitor
 	private $username;
 
 
-	/**
-	 * Class constructor
-	 *
-	 * @return	void
-	 * @param	string $URL						The base URL of your CreateSend site. e.g. http://example.createsend.com/.
-	 * @param	string $username				The username you use to login to Campaign Monitor.
-	 * @param	string $password				The password you use to login to Campaign Monitor.
+    /**
+     * Class constructor
```

Comment:
this is a bit of a useless comment, could be removed but I don't think this is a blocker